doc/release-notes.md

```diff
@@ -20,361 +22,179 @@ How to Upgrade
 If you are running an older version, shut it down. Wait until it has completely
 shut down (which might take a few minutes for older versions), then run the
 installer (on Windows) or just copy over /Applications/Dash-Qt (on Mac) or
-dashd/dash-qt (on Linux).
+dashd/dash-qt (on Linux). Because of per-UTXO fix (see below) there is a
+one-time database upgrade operation, so expect a slightly longer first run.
 
 Downgrade warning
 -----------------
-### Downgrade to a version < 0.12.2
 
-Because release 0.12.2 includes DIP0001 (2 MB block size hardfork) plus
-a transaction fee reduction and a fix for masternode rank calculation algo
-(which activation also depends on DIP0001) this release will not be
-backwards compatible after DIP0001 lock in/activation happens.
+### Downgrade to a version < 0.12.2.2
+
+Because release 0.12.2.2 includes per-UTXO fix (see below) which changes the
+structure of the internal database, this release is not backwards compatible.
+You will have to reindex database if you decide to use previous version.
 
 This does not affect wallet forward or backward compatibility.
 
+
 Notable changes
 ===============
 
-DIP0001
--------
-
-We outline an initial scaling mechanism for Dash. After deployment and activation, Dash will be able to handle double the transactions it can currently handle. Together with the faster block times, Dash we will be prepared to handle eight times the traffic of Bitcoin.
-
-https://github.com/dashpay/dips/blob/master/dip-0001.md
-
+Per-UTXO fix
+------------
 
-Fee reduction
--------------
+This fixes potential vulnerability, so called 'Corebleed', which was
+demonstrated this summer on Вrеаkіng Віtсоіn Соnfеrеnсе іn Раrіs. The DoS can
+cause nodes to allocate excessive amounts of memory, which leads them to a halt.
+You can read more about the fix in the original Bitcoin Core pull request
+https://github.com/bitcoin/bitcoin/pull/10195
 
-All transaction fees are reduced 10x (from 10K per Kb to 1K per Kb), including fees for InstantSend (from 0.001 DASH per input to 0.0001 per input)
+To fix this issue in Dash Core however, we had to backport a lot of other
+improvements from Bitcoin Core, see full list of backports in the detailed
+change log below.
```

Comment:
**changelog** (no space)

Reply:
Not sure about this one, both forms are legit imo e.g. https://www.gnu.org/prep/standards/html_node/Change-Logs.html but http://keepachangelog.com/en/1.0.0/